src/Data/Vec/Properties.agda

```diff
@@ -1130,6 +1182,29 @@ toList∘fromList : (xs : List A) → toList (fromList xs) ≡ xs
 toList∘fromList List.[]       = refl
 toList∘fromList (x List.∷ xs) = cong (x List.∷_) (toList∘fromList xs)
 
+toList-cast : ∀ .(eq : m ≡ n) (xs : Vec A m) → toList (cast eq xs) ≡ toList xs
```

Comment:
Same remark as for `fromList-cast`: but this one, I think, can be analysed in terms of `cong toList`? or not here, either?

Reply:
I don't think it can.  `cast eq xs` and `xs` have types `Vec A m` and `Vec A n` where `m` and `n` are only propositionally, not definitionally equal, so `cong toList` won't type-check here.

Reply:
D'oh! Suggest perhaps relativising instead to:
```agda
toList-cast : ∀ .(eq : m ≡ n) (xs : Vec A m) {ys} → cast eq xs ≡ ys toList (cast eq xs) ≡ toList ys
```
and similarly for `fromList-cast`? or does this suffer from the same problem (you'd think I could think this through myself, but... brain-fog/code-blindness)?

Reply:
:man_facepalming:

Reply:
FWIW, `toList-cast` was introduced in #1839. I only moved the definition down so that `fromList` properties and `toList` properties are grouped together.